Section 1: Introduction and Setup/4-RoR Development Environment/testing_project/app/javascript/packs/application.js

```diff
@@ -0,0 +1,13 @@
+// This file is automatically compiled by Webpack, along with any other files
```

Comment:
what is this file for?

Reply:
`application.js` is where you include all the Javascript scripts that you needed for your site.

Reply:
1. `include` How?